sdk/servicebus/service-bus/src/session/messageSession.ts

```diff
@@ -972,7 +967,7 @@ export class MessageSession extends LinkEntity {
           "[%s] Batching Receiver '%s'  max wait time in seconds %d over.",
           this._context.namespace.connectionId,
           this.name,
-          maxWaitTimeInSeconds
+          maxWaitTimeInMs / 1000
```

Comment:
Same comment here, let's log out the milliseconds instead of dividing.

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/8169/commits/1313eb66765b5cc1fe2db2ce12c335d842f79db4